staff/jordi-relano/bash/README.me

```diff
@@ -0,0 +1,49 @@
+```bash										
+las comillas invertidas generan un espacio cuando tabulamos										
```

Comment:
Organize and follow the example of Manu's file https://drive.google.com/drive/folders/1zIyeQgKR3nIPAuaQ5jA3Av5h9MHprHxd and remember to write everything in english 🧐